### PR TITLE
Create LICENSE.txt

### PR DESCRIPTION
**Pull request by Medha-B**
Apache License, version 2.0.

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (kaitoii11) [inline LICENSE.txt]: Add the year and owner here

**Review by kaitoii11 — CHANGES_REQUESTED** [review]

**Comment by kaitoii11** [comment]
Add the License section in the README as well.

File: LICENSE.txt (hunk 1 above)
Comment (Medha-B) [reply LICENSE.txt]: Who is the owner though?

Comment (kaitoii11) [reply LICENSE.txt]: I guess you can put the name for all 4 of us.

**Review by kaitoii11 — COMMENTED** [review]
LGTM

**Pull request merged**